clevercloud/functions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -l
+
+set -o errexit -o nounset -o xtrace
+
+function install_dependencies() {
+    mkdir -p ${HOME}/.local/bin
+
+    # Download and extract kopia
+    wget https://github.com/kopia/kopia/releases/download/v0.15.0/kopia-0.15.0-linux-x64.tar.gz
```

Comment:
```suggestion
    wget https://github.com/kopia/kopia/releases/download/v0.15.0/kopia-0.15.0-linux-x64.tar.gz
```

<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ We want to lock the version to `0.15.0` ?